### ISSUE TITLE
heat capacitator node setpoint question

### ISSUE THREAD

**Issue by Swoop86** [ottopaulsen/node-red-contrib-power-saver#142]
What is the purpose of the "setpoint" being injected into the node? It seems to get the value from what is configured inside the node config of heat capacitator. It makes it difficult if one wants to change the temperature.

**Comment by TomTorger** [comment]
The setpoint is the "ideal temperature", that is what you would like the mean temperature to be. A bit simplified: The node will add a dT in cheap periods, and subtract dT in expensive periods to this setpoint

**Comment by Swoop86** [comment]
Which of the two is the one that is being used, the one defined inside the node or the setpoint injected? (Per documentation it is not explained)
![E156335D-F7CE-4954-AD68-0116C8F5C3D0](https://user-images.githubusercontent.com/5503114/210132006-6446d598-cfcc-468f-858d-e8a5c630b924.jpeg)

**Comment by TomTorger** [comment]
This is in Norwegian, but I guess you'll be able to deal with that :-)
The line in the middle (the middle temperature). The procedure to configure the node is as follows:

1. Decide on a setpoint (the ideal temperature)
2. Decide on how far away from the setpoint the temperature is allowed to move (max temp adjustment)
3. Specify how much time is required to increase/decrease the temperature by 1C under max/no load.
4. Add "heating boost" and "cooling boost" if you need to trigger controllers (adds this temperature in the heating/cooling periods to assert that the unit is at full/no load)

![image](https://user-images.githubusercontent.com/49571281/210154718-758d9707-71cd-4191-955a-7fa319d9a736.png)

**Comment by Swoop86** [comment]
Sent you a message on messenger :)

Just to clarify my question, in the documentation example it is specified that an input_number should be provided (setpoint) that tells the node the ideal temperature. This setpoint is also being set inside the node, thus it seems that the input_number is never being used as it is the values from inside the node that are being used :)

**Comment by TomTorger** [comment]
Yes, that is true :-) When the setpoint is provided dynamically, the value in the form is overwritten. The NodeRed front-end supplies the values in the form above to the back-end upon deployment (when deploying the flow), and I have not seen any write-back functionality to update these... In the example below, when the dynamic configuration message is received, the initial setpoint values is overwritten... I will have a look at the formulation in the documentation, I might not have made that clear (It is on my list to update a lot of the documentaiton :-) )

![image](https://user-images.githubusercontent.com/49571281/210455320-9ee90681-5e91-408f-ba86-73401d443494.png)

**Comment by mariuschr** [comment]
I have an issue regarding the value placed in the form. I don't know what triggers it, but it's probably related to reboots, or that Node-RED restarts, or something like that.

I have an automation in HA which sends the correct setpoint based on if I'm at the cabin or not. If I deploy the automation in Node-RED, the setpoint changes in Node-RED, but not in HA.

I have tried making automations in HA to confirm that the right value is stored in Node-RED, but seemingly this value can't be extracted. Making the automation that sends the right setpoint to Node-RED which runs every x minutes doesn't help, as the HA setpoint hasn't changed, and therefore nothing is being sent to Node-RED.

Currently I'm solving it by changing the setpoint temperature up and down, but it's not optimal.

Any tips?
